### ISSUE TITLE
FormattedLLMCall.forward() returns a Variable with a default requires_grad = True?

### ISSUE THREAD

**Issue by HelloWorld4747** [zou-group/textgrad#208]
Hello,

First of all, thank you so much for sharing such awesome work!

I am currently defining my own loss function with the FormattedLLMCall operation (exactly same as how you did in your demo code `Tutorial-Test-Time-Loss-for-Code.ipynb'). I am a bit confused about how come the attribute "required_grad" of the returned loss (a Variable coming from FormattedLLMCall.forward()) is set with a default value "True". Currently, it seems no effect even though the 'required_grad' is set to 'True'. But, shouldn't this instance attribute ("required_grad") set to "True", if and only if we want to optimize the corresponsding Variable? I am more curious about your idea when developing this FormattedLLMCall class.

Looking forward to your response!
Cheers!
